semantic-convention-utils/build.gradle.kts

```diff
@@ -1,7 +1,10 @@
 plugins {
     `java-library`
+    id("org.hypertrace.publish-plugin")
 }
 
+group = "org.hypertrace.utils"
```

Comment:
```suggestion
group = "org.hypertrace.semanticconventionutils"
```

Reply:
or `semconutils` ?

Reply:
This `org.hypertrace.utils` is very generic. Pl. add appropriate prefix.

Reply:
`org.hypertrace.utils` this is the group id right? 
Consumers can use it like: `org.hypertrace.utils:semantic-convention-utils:<version>`